src/managers/MessageManager.js

```diff
@@ -62,8 +63,8 @@ class MessageManager extends BaseManager {
    *   .then(messages => console.log(`${messages.filter(m => m.author.id === '84484653687267328').size} messages`))
    *   .catch(console.error);
    */
-  fetch(message, cache = true) {
-    return typeof message === 'string' ? this._fetchId(message, cache) : this._fetchMany(message, cache);
+  fetch(message, cache = true, force = true) {
```

Comment:
This default (`true`) does not match the default in the JSDocs (line 48, documented as `false`).